target/classes/application.properties

```
@@ -1,4 +1,7 @@
 spring.datasource.url=jdbc:postgresql://localhost:5432/telegram_not_bot
 spring.datasource.username=telegram_not_bot
 spring.datasource.password=bot123
-telegram.bot.token=6314365409:AAEmdZutbXfmuTCKDN7KVOkKfA8SwgB5cUs
+telegram.bot.token=6411648337:AAEF6AWvhX7djsU_3OXpcjUG804OdMnjlVk
+spring.jpa.hibernate.ddl-auto= update
```

```suggestion
spring.jpa.hibernate.ddl-auto=validate
```
таблицы создаём через liquibase, поэтому тут должно быть validate